broker-daemon/state-machines/order-state-machine.js

```diff
@@ -42,6 +43,9 @@ const OrderStateMachine = StateMachine.factory({
     new StateMachineLogging({
       skipTransitions: [ 'goto' ]
     }),
+    new StateMachineDates({
```

Comment:
I'm assuming that this plugin needs to be added before `StateMachinePersistence` in order to make sure that the date of the latest transition is added before being saved to the db.

If that's the case, it may be worth testing that behavior, or at least documenting the requirement.

Reply:
Hm, I haven't found it to be an issue. All the dates get saved properly 🤔 but I'll do some digging.